src/map/status.c

```diff
@@ -9868,6 +9875,17 @@ int status_change_start(struct block_list *src, struct block_list *bl, enum sc_t
 		}
 		calc_flag&=~SCB_DYE;
 	}
+	/* Currently No SC's use this
```

Comment:
I'd prefer an `#if 0` / `#endif` to (temporarily) disable code, because it nests correctly